controls/radgridview/export/excel/export-with-spreadsheetstreamingexport.md

```diff
@@ -89,6 +89,8 @@ Via the GridViewSpreadStreamExportOptions you can customize how the RadGridView
 - **HiddenColumnExportOption**: Gets or sets a value indicating how hidden columns are exported. The **HiddenColumnExportOptions** enumeration provides three possible options: **ExportAlways**, **DoNotExport** and **ExportAsHidden**.
 - **ColumnWidth**: Gets or sets the width of the columns that are exported.
 - **Items**: The collection of items to be exported.
+- **ShowGroupRows**: Specifies whether the groups' items should be exported (this property was introduced in **R3 2019**).

```

Comment:
I think its better to say "Specifies whether the group rows should be included on export".